contrib/node/install-sdn.sh

```diff
@@ -1,36 +1,31 @@
 #!/bin/bash
+set -x
 
 os::provision::install-sdn() {
-  local default_target="/usr"
-
   local deployed_root=$1
-  local target=${2:-${default_target}}
+  local binaries_path=$2
+  local target=${3:-}
+  local target_usrdir="${target}/usr"
+  local target_bindir="${target_usrdir}/bin"
+  local target_confdir="${target}/etc/cni/net.d"
+  local target_cnidir="${target}/opt/cni/bin"
 
-  if [ ! -d ${target} ]; then
-    mkdir -p ${target}
-  fi
+  mkdir -p -m 0755 "${target_usrdir}"
```

Comment:
@stevekuznetsov I'm OK changing this, but this format is used nowhere else in OpenShift scripts that I can see, nor in origin.spec, nor in Golang calls like os.Mkdir() and os.Chmod().  We can certainly blaze the trail here, at the cost of inconsistency.